### PR TITLE
add basic design for existing components

### PR DESCRIPTION
**Pull request by Arthour**
- add MUI
- add design for existing components (including header)
- add support for light/dark mode
image

### CODE REVIEW

File: frontend/src/index.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
```
Comment (l1f) [inline frontend/src/index.tsx:3]: webstorm seems to turn quotes into single quotes as well :(

**Comment by l1f** [comment]
LGTM. (All that bothers me a bit is that we use different code formatters :D)

File: frontend/src/index.tsx (hunk 1 above)
Comment (Arthour) [reply frontend/src/index.tsx]: just added tslint & .editorconfig so we both use the same coding conventions
so this shouldn't be a problem in the future anymore

Comment (l1f) [reply frontend/src/index.tsx]: Would you adjust the quotation mark? :D

**Pull request merged**